script/intentfest/sentenceParser.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+"""Parse sentences using language intent files."""
+from __future__ import annotations
+
+import json
+from typing import Any, Dict
+
+import yaml
+from hassil.intents import Intents
+from hassil.recognize import recognize
+from hassil.util import merge_dict, normalize_whitespace
+
+from .const import SENTENCE_DIR, TESTS_DIR
+from .util import get_slot_lists, load_merged_responses, render_response
+
+
+class SentenceParser:
+    def __init__(self, language) -> None:
+        self.slot_lists = None
+        self.intents = None
+        self.responses = None
+
+        self.output_dict = None
+
+        self.prepareNewIntents(language)
+
+    def prepareNewIntents(self, language):
+        language_dir = SENTENCE_DIR / language
+        tests_dir = TESTS_DIR / language
+
+        # Load test areas and entities for language
+        test_names = yaml.safe_load((tests_dir / "_fixtures.yaml").read_text())
+        self.slot_lists = get_slot_lists(test_names)
+
+        # Load intents
+        intents_dict: Dict[str, Any] = {}
+        for intent_path in language_dir.glob("*.yaml"):
+            with open(intent_path, "r", encoding="utf-8") as intent_file:
+                merge_dict(intents_dict, yaml.safe_load(intent_file))
```

Comment:
a) these are sentences not intents
b) we have a util function for this `load_merged_sentences`

Reply:
Ok, will use the `load_merged_sentences` function, it's written that way because I relied on the code available here [script/intentfest/parse.py](#diff-ab0ef75141f7639218d09a8b8442c9e33ae6063bde7c5f80d63c0e94c4260c98.)

Reply:
Now I'm using `load_merged_sentences`, thanks!